docs/admin/install.rst

```diff
@@ -1341,8 +1352,8 @@ The following configuration runs Weblate as WSGI, you need to have enabled
 
 .. note::
 
-    Weblate requires Python 3, so please make sure you are running Python 3
-    variant of the modwsgi. Usually it is available as a separate package, for
+    Weblate requires Python 3, so please ensure you are running the Python 3
+    variant of mod_wsgi. Usually it is available as a separate package, for
     example ``libapache2-mod-wsgi-py3``.
 
 .. seealso::
```

Comment:
"Running Weblate under path" means what?
The $path set for Weblate?

Reply:
When Weblate is available on `https://example.com/weblate/`.